packages/ff-pbt/ff-pbt.0.5.0/opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "2.0"
+synopsis:
+  "Property based testing library for finite fields over the package ff-sig"
+description:
+  "Property based testing library for finite fields over the package ff-sig"
+maintainer: "Danny Willems <be.danny.willems@gmail.com>"
+authors: "Danny Willems <be.danny.willems@gmail.com>"
+license: "MIT"
+homepage: "https://gitlab.com/dannywillems/ocaml-ff"
+bug-reports: "https://gitlab.com/dannywillems/ocaml-ff/issues"
+depends: [
+  "ocaml" {>= "4.08"}
+  "dune" {>= "2.0"}
+  "zarith" {>= "1.9.1" & < "2.0"}
+  "ff-sig" {= "0.5.0"}
```

Comment:
Note that this can be replaced by `"ff-sig" {= version}`. It's probably more maintainable for you and for us.

Reply:
Thanks for the information, I didn't know.